website/content/docs/secrets/databases/oracle.mdx

```diff
@@ -1,37 +1,60 @@
+page_title: Oracle - database - secrets engines
 description: |-
   Oracle is one of the supported plugins for the database secrets engine. This
   plugin generates database credentials dynamically based on configured roles
   for the Oracle database.
 ---
 
-# Oracle Database Secrets Engine
+# Oracle database secrets engine
 
-This secrets engine is a part of the Database Secrets Engine. If you have not read the
-[Database Backend](/vault/docs/secrets/databases) page, please do so now as it explains how to set up the database backend and
+-> The Oracle database plugin is now available for use with the database secrets engine in HCP Vault.
+   The plugin configuration (including installation of the Oracle Instant Client library) is managed
+   by HCP. Refer to the HCP Vault tab for more information.
+
+This secrets engine is a part of the database secrets engine. If you have not read the
+[database backend](/vault/docs/secrets/databases) page, please do so now as it explains how to set up the database backend and
 gives an overview of how the engine functions.
 
 Oracle is one of the supported plugins for the database secrets engine. It is capable of dynamically generating
-credentials based on configured roles for Oracle databases. It also supports [Static Roles](/vault/docs/secrets/databases#static-roles).
+credentials based on configured roles for Oracle databases. It also supports [static roles](/vault/docs/secrets/databases#static-roles).
+
+## Capabilities
+
+<Tabs>
+<Tab heading="Vault" group="vault">
 
 ~> The Oracle database plugin is not bundled in the core Vault code tree and can be
 found at its own git repository here:
 [hashicorp/vault-plugin-database-oracle](https://github.com/hashicorp/vault-plugin-database-oracle)
 
 ~> This plugin is not compatible with Alpine Linux out of the box.
 
-## Capabilities
-
 | Plugin Name                                                          | Root Credential Rotation | Dynamic Roles | Static Roles | Username Customization |
 | -------------------------------------------------------------------- | ------------------------ | ------------- | ------------ | ---------------------- |
 | Customizable (see: [Custom Plugins](/vault/docs/secrets/databases/custom)) | Yes                      | Yes           | Yes          | Yes (1.7+)             |
 
+</Tab>
+<Tab heading="HCP Vault" group="hcp">
+
+~> The Oracle Database Plugin is managed by the HCP platform. No extra installation steps are required for HCP Vault.
+
+| Plugin Name                                                          | Root Credential Rotation | Dynamic Roles | Static Roles | Username Customization |
+| -------------------------------------------------------------------- | ------------------------ | ------------- | ------------ | ---------------------- |
+| `vault-plugin-database-oracle`                                       | Yes                      | Yes           | Yes          | Yes (1.7+)             |
```

Comment:
can for the HCP group we also mention the plugin version? 0.8.1

Reply:
How often does this plugin version change? I don't see any other reference to versions on this page. We would then need to update the page every time the version changes.

Reply:
yeah, fair callout. I am not sure, but we should tell this information as folks might reach out to us if better features are in a newer release. it's ok to take it out here.

Reply:
Would the version of the plugin be better served in the cloud docs? Maybe in the comparison table?

Reply:
works for me as long as we document it somewhere. Can we also considering making the section External Plugins so that we don't have to comparison table for all internal plugins.